tests/test_cirq_device.py

```diff
@@ -156,6 +156,22 @@ def test_outer_init_of_qubits_with_wire_label_ints(self):
             dev.map_wires(Wires(label)) == Wires(idx) for label, idx in zip(user_labels, sort_order)
         )
 
+    @pytest.mark.parametrize(
+        "op,expected",
+        [
+            (qml.PauliX, True),
+            ((qml.PauliX(0)**1.1).name, True),
+            ("PauliX**1.1", True),
+            (qml.IsingXX, False),
+            ("IsingXX", False)
+        ],
```

Comment:
Why are we adding a test for:
- `qml.PauliX`
- `qml.IsingXX`
These were already supported, right? They were not tested?